manual_test_websocket/main/test/e2e/ws.e2e.js

```diff
@@ -313,6 +314,7 @@ describe('serverless', () => {
       const callInfo = JSON.parse(await c.ws.receive1());
       now = Date.now();
       expectedCallInfo = { event:{ body: '{"action":"getCallInfo"}', ...createExpectedEvent(c.id, 'getCallInfo', 'MESSAGE', callInfo.info.event) }, context:createExpectedContext(callInfo.info.context) };
+      delete callInfo.info.context; delete expectedCallInfo.context; // Not checking context. Relying on it to be correct because serverless-offline uses general lambda context method
```

Comment:
Please lint. A new line per statement.

Reply:
How do you lint?
I find that I can `npm run lint` only on the root project without any errors.

Reply:
I checked and eslint checks `main/test/e2e/ws.e2e.js` but it doesn't mind "A new line per statement.". 
If it did mind, this PR would not pass CI.
Maybe you have a different lint configuration, i.e., a local version of `eslint-config-dherault`.

Reply:
You have to delete the eslint ignore. Try to rebase your PR.

Reply:
> You have to delete the eslint ignore. 

`.eslintignore` contains only `**/node_modules`. We wouldn't won't to check all the modules, right?
> Try to rebase your PR.

I cloned  #711 to a new directory and running `npm run lint` there dosn't show any error. If I intorduce an error in `main/test/e2e/ws.e2e.js`, running `npm run lint` does catch the error.

Any other idea?

Reply:
Ok Let's go with the current implementation then :)